docs/DeveloperGuide.md

```diff
@@ -1133,26 +1199,49 @@ testers are expected to do more *exploratory* testing.
 
 { end of `manual_testing#launch_and_shutdown` written by: __________ }
 
-{ start of `manual_testing#adding_a_student` written by: __________ }
+{ start of `manual_testing#adding_a_student` written by: Ngoh Wei Yue }
 
 ### 11.2. Adding a student
 
-1. Deleting a student while all students are being shown
-
-  1. Prerequisites: List all students using the `list` command. Multiple students in the list.
-
-  1. Test case: `delete 1`<br>
-     Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
-
-  1. Test case: `delete 0`<br>
-     Expected: No student is deleted. Error details shown in the status message. Status bar remains the same.
-
-  1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+  1. Adding a student while all students are being shown.
+  
+  1. Prerequisites: List all students using the list command. No students with the same matriculation number or NUS email address as any of the students to be added during testing.
+
+  1. Test case: `addstu n/Alvin Boon m/A0123456X e/alvinboon@u.nus.edu`<br>
+     Expected: A student is added to the bottom of the student list. A success message including the particulars of the added student is shown in the result box.
+     
+  1. Test case: `addstu n/Cathy Duigan m/A1123456X e/cathyduigan@u.nus.edu t/helpful`<br>
+       Expected: Similar to previous.
+       
+  1. Test case: `addstu m/A2123456X n/Elbert Foo e/elbertfoo@u.nus.edu`<br>
+       Expected: Similar to previous.
+       
+  1. Test case: `addstu n/Gina Ho m/A3123456X`<br>
+     Expected: No student is added. Error message indicating an invalid command format is shown in the result box.
+     
+  1. Test case: `addstu n/Gina Ho e/ginaho@u.nus.edu`<br>
```

Comment:
Maybe pictures could be provided for at least one positive test case and one negative test case each?

Reply:
Nice suggestion! Will do!